web3-dev/smart-contracts/sc-execution.rst

```diff
@@ -0,0 +1,66 @@
+========================
+Smart contract execution
+========================
+
+Introduction
+============
+
+In Massa you can execute arbitrary smart contracts, you are not limited to deploy and then call the smart contract.
+There is different possibilities to execute smart contracts in different contexts.
+This documentation will explain you the different possibilities and how to use them.
+You can choose the one that match the most your usecase.
+
+There is examples for the differents ways to execute smart contracts available
+in the smart contract sdk `massa-web3 <https://github.com/massalabs/massa-web3>`_.
+
+Differents ways to execute smart contracts
+==========================================
+
+Operation execute smart contract
+--------------------------------
+
+This operation allow you to send a smart contract that will be executed on the blockchain
+(not stored) and can perform all the state changes as a normal smart contract.
+It's often the best way to deploy an other smart contract using the operation datastore of the execute operation.
+Our `deployer <https://github.com/massalabs/massa-sc-toolkit/blob/main/packages/sc-deployer/src/index.ts>`_
+uses this operation to deploy smart contracts.
```

Comment:
This needs to be rewritten. I don't fully understand this sentence: 

''It's often the best way to deploy an other smart contract using the operation datastore of the execute operation.'' -> 
Why? - do I save gas this way? 
Why is it better? 
Do we explain operation datastore before this paragraph/section?

Reply:
Yes it's better because it use less gas and it's more flexible to deploy multiple smart-contracts.

There is no current docs on operation datastore we should add one page on this